app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+
+    if len(split_command) != 3 or split_command[0] != "cp":
+        print("Invalid command")
+        return
+
+    file_to_copy = split_command[1]
+    copied_file = split_command[2]
+
+    if file_to_copy == copied_file:
+        return
+
+    with open(file_to_copy, "r") as file, open(copied_file, "w") as copy:
```

Comment:
Consider adding error handling for file operations. If the file to copy does not exist or there are permission issues, the program will raise an exception. Using a try-except block can help manage these potential errors gracefully.